aspnetcore/includes/aspnetcore/includes/net-core-prereqs-all-2.2.md

```diff
@@ -0,0 +1,17 @@
+# [Visual Studio](#tab/visual-studio)
+
+* [Visual Studio 2017 version 15.9 or later](https://visualstudio.microsoft.com/downloads/) with the **ASP.NET and web development** workload
+* [.NET Core SDK 2.2 or later](https://www.microsoft.com/net/download/all)
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+* [Visual Studio Code](https://code.visualstudio.com/download)
+* [.NET Core SDK 2.2 or later](https://www.microsoft.com/net/download/all)
+* [C# for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
```

Comment:
Not all tutorials include Mac sections -- we should have a ...all-no-mac or ...vs-vsc version.

Reply:
I agree. We should create _net-core-prereqs-vs-vsc-2.2.md_ with VS and VSC and then net-core-prereqs-all-2.2.md include would contain _net-core-prereqs-vs-vsc-2.2.md_ and the VS mac content.

Reply:
Adding the heading sounds good to me.